Loop.xcconfig

```diff
@@ -42,6 +42,9 @@ LOOP_PROVISIONING_PROFILE_SPECIFIER_STATUS_EXTENSION_RELEASE =
 LOOP_PROVISIONING_PROFILE_SPECIFIER_WATCHAPP_RELEASE =
 LOOP_PROVISIONING_PROFILE_SPECIFIER_WATCHAPP_EXTENSION_RELEASE =
 
+// Min iOS Version [DEFAULT]					
+IPHONEOS_DEPLOYMENT_TARGET = 14.1
+
```

Comment:
PR title says bumping to min version to 15.0. Curious if this should be 14.1?

Reply:
Ah, sorry if this is not obvious: 14.1 is what the default min version is today.  This specifies that for the DIY builds, effectively, and it is overridden by the [LW PR](https://github.com/tidepool-org/LoopWorkspace/pull/748).

Reply:
I assume this will just be used for DIY, and will be overridden in our builds at a higher level.

Reply:
@ps2: Exactly.